index.html

```diff
@@ -10,10 +10,54 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
-
+ <header>
+    <a class="logo" href="index.html"><img src="img/karma-logo.svg" alt="karma logo"></a>
+    <nav>
+        <ul class="list">
+            <li class="items"><a class="links" href="#"> Meet Karma</a></li>
+            <li class="items"><a class="links" href="#"> How it Works</a></li>
+            <li class="items"><a class="links" href="#"> Store</a></li>
+            <li class="items"><a class="links" href="#"> Blog</a></li>
+            <li class="items"><a class="links" href="#"> Help</a></li>
+            <li class="items"><a class="links" href="#"> Login</a></li>  
+        </ul>     
+    </nav>
+ </header> 
+ <main> 
+    <section class="p1">
+        <h1>Introducing Karma</h1>
+        <h4>Bring WiFi with you, everywhere you go.</h4>
+        <button><a href="#">Learn More</a></button>
+    </section>
+    <section>
+        <h2>
+            Everyone needs alitte Karma.
+        </h2>
+        <div class="features">
+            <div class="p2">
```

Comment:
Nearly all your CSS classnames are really clear and easy to understand. Maybe worth changing `p1` and `p2` as well so when you see them in the CSS you'll remember what they're for.